package.json

```diff
@@ -45,7 +45,7 @@
     ]
   },
   "dependencies": {
-    "@shopgate/cloud-sdk-webpack": "^1.5.4",
+    "@shopgate/cloud-sdk-webpack": "^1.6.0",
```

Comment:
Nice try ;) But this has to be done via `npm i` when the package is really updated! Please revert this change here!